testdata/helm/memcached-operator/bundle/manifests/cache.example.com_memcacheds.yaml

```diff
@@ -1,50 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why are a lot of the files under the `manifests/` directory deleted for the helm testdata?

Reply:
Updated test data properly.